### PR TITLE
Handle missing onRemediationCreated fn

### PR DESCRIPTION
**Pull request by fhlavac**
Handle undefined onRemediationCreated function

### CODE REVIEW

**Review by Hyperkid123 — APPROVED** [review]

File: packages/remediations/src/utils/utils.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ export const submitRemediation = (formValues, data, basePath, setState) => {
     api.patchRemediation(id, { add, auto_reboot: formValues[AUTO_REBOOT] }, basePath) ||
     api.createRemediation({ name: formValues[SELECT_PLAYBOOK], add, auto_reboot: formValues[AUTO_REBOOT] }, basePath)).then(({ id }) => {
         setState({ id, percent: 100 });
-        data.onRemediationCreated({
+        data.onRemediationCreated && data.onRemediationCreated({
```
Comment (karelhala) [inline packages/remediations/src/utils/utils.js:159]: Optional chaining mught be better.
```suggestion
        data?.onRemediationCreated({
```

Comment (fhlavac) [reply packages/remediations/src/utils/utils.js:159]: @karelhala is it really applicable in this case with a function?

Comment (Hyperkid123) [reply packages/remediations/src/utils/utils.js:159]: @karelhala that does not work on functions

Comment (karelhala) [reply packages/remediations/src/utils/utils.js:159]: Oh, I forgot the other conditonal one as well
```suggestion
        data?.onRemediationCreated?.({
```

Comment (fhlavac) [reply packages/remediations/src/utils/utils.js]: @karelhala oh, great! Didn't know it works like this as well, thank you :-)

**Pull request merged**